File: equo-webwrapper/src/main/java/samplename/SampleNameApplication.java
package samplename;

import com.equo.application.api.IEquoApplication;
import com.equo.application.model.EquoApplicationBuilder;
import com.equo.dev.EquoApp;
import org.osgi.service.component.annotations.Component;

@Component
public class SampleNameApplication implements IEquoApplication {

    public static void main(String[] args) throws Exception {
        EquoApp.launch();
    }

    public EquoApplicationBuilder buildApp(EquoApplicationBuilder appBuilder) {
        return appBuilder
                .wrap("sampleurl")
                .start();
    }
}

File: kotlin/src/main/kotlin/samplename/SampleNameApplication.kt
package samplename

import com.equo.application.api.IEquoApplication
import com.equo.application.model.EquoApplicationBuilder
import com.equo.dev.EquoApp
import org.osgi.service.component.annotations.Component

fun main(args: Array<String>) {
    EquoApp.launch()
}

@Component
class SampleNameApplication : IEquoApplication {
    override fun buildApp(appBuilder: EquoApplicationBuilder): EquoApplicationBuilder {
        return appBuilder
            .withUI("index.html")
            .start()
    }
}
